src/jit/compiler.cpp

```diff
@@ -5589,7 +5596,7 @@ void Compiler::compCompileFinish()
     {
         if (compJitHaltMethod())
         {
-#if !defined(_TARGET_ARM64_) && !defined(_HOST_UNIX_)
+#if !defined(_TARGET_ARM64_) && !defined(_TARGET_ARM_) && !defined(_HOST_UNIX_)
```

Comment:
This doesn't look related.  I'd prefer it be in its own PR, but if you want to keep it here, please at least note it in the description of the change.